assignments/can_drive/can_drive.js

```diff
@@ -1,14 +1,19 @@
 function CanDrive(hasDrivingLiscence, isTired, isSober) {
     // Write you code here
-    if(hasDrivingLiscence==false || isTired==false || isSober==true) {
-        return String("You cannot drive")
+    if(hasDrivingLiscence === false){
+        return 'You cannot drive'
     }
-    else if((hasDrivingLiscence==true || isTired==false || isSober==true)){
-        return String("You can drive")
+    else if(isTired === true){
```

Comment:
The code is correct ,a small change is that we can write these two else if conditions in a single condition. This just helps to understand the code better with less conditional statements

Reply:
@AjayKomireddy 

OK Ajay 
I will change the code.


Thank You.

Reply:
@AjayKomireddy 

Hai Ajay, I have changed my code as per requirements.


Please Check Assignment Once. 


Thank You.